### PR TITLE
Add Babel in Meteor instructions

### PR DESCRIPTION
**Pull request by grigio**

### CODE REVIEW

File: docs/using-babel.md [hunk 1]
```diff
@@ -74,6 +74,26 @@ require("babel/register");
...
 </blockquote>
 
+## Meteor
+
+<blockquote class="babel-callout babel-callout-info">
+  <p>
+    <em>Issues should be reported on the <a href="https://atmospherejs.com/grigio/babel">grigio:babel</a> issue
+    <a href="https://github.com/grigio/meteor-babel/issues">tracker</a></em>.
+  </p>
+</blockquote>
```
Comment (thejameskyle) [inline docs/using-babel.md]: Can you replace this section with the following:
```js
<blockquote class="babel-callout babel-callout-info">
  <p>
    See <strong>meteor-babel</strong>'s
    <a href="https://github.com/grigio/meteor-babel">repo</a> for more
    info. If you find any bugs please
    <a href="https://github.com/grigio/meteor-babel/issues">report
    them</a>.
  </p>
  <p>
    <em>Issues with the output should be reported on the babel
    <a href="https://github.com/babel/babel/issues">issue tracker</a></em>.
  </p>
</blockquote>
```

File: docs/using-babel.md [hunk 2]
```diff
@@ -74,6 +74,26 @@ require("babel/register");
...
+**Install**
+
+Inside your Meteor app directory
+
+```sh
+$ meteor add grigio:babel
+```
+**Usage**
+
+Any supported extension will be precompiled in ES5 by babel. Some features may be not available, here [some tests](https://github.com/grigio/meteor-babel/blob/master/tests/basic_test.es6.js) of the supported features in the [Meteor](https://www.meteor.com/) platform.
```
Comment (thejameskyle) [inline docs/using-babel.md]:
```md
**Usage**

Any files with these extensions will be precompiled with Babel: `.es6.js`, `.es6`, `.es`, `.jsx`.
```

**Comment by grigio** [comment]
@thejameskyle news?

**Comment by thejameskyle** [comment]
Sorry, I never was notified that you updated this.

**Pull request merged**